test/langtools/jdk/javadoc/doclet/testClassTree/TestClassTree.java

```diff
@@ -43,6 +45,37 @@ public static void main(String... args) throws Exception {
         tester.runTests();
     }
 
+    @Test
+    public void testBadPkg() {
+        // Given badpkg package containing class ChildClass with an undefined
+        //       base class, implementing undefined interface and a defined
+        //       interface
+        // When  the javadoc is generated with --ignore-source-errors option
+        javadoc("--ignore-source-errors",
+                "-d", "badout",
+                "--no-platform-links",
+                "-sourcepath", testSrc,
+                "badpkg");
+        // Then javadoc exits successfully
+        checkExit(Exit.OK);
+        // And generates html for the ChildClass
+        checkOutput("badpkg/package-tree.html", true,
+        """
+        <li class="circle">badpkg.<a href="ChildClass.html" class="type-name-link" \
+        title="class in badpkg">ChildClass</a> (implements java.lang.Iterable&lt;T&gt;)</li>
+        """);
```

Comment:
arguments, like this text block, should be indented.

In general, if you're using an IDE that can reformat fragments of code, like a method call, it should be enough/OK to use the IDE reformat calls like this.